proto/viam/app/v1/app.proto

```diff
@@ -377,6 +377,7 @@ message Organization {
   // associated with this org will be stored in this region.
   string default_region = 5;
   optional string cid = 6;
+  bool has_synced_data = 7;
```

Comment:
We don't necessarily need to update the API in order to add a new field to docs in the organizations collection. In fact, if we were to add it, I would say that this feels like a field that users don't need to know about.

Either way, going this route would require a data migration. Making sure this is a step that you're aware of?